### ISSUE TITLE
Move themes/ folder

### ISSUE THREAD

**Issue by devyanlab** [teepluss/laravel-theme#105]
Is is possible to move themes/ folder outside public/?
Is it possible to move it to resources/?

**Comment by kayrules** [comment]
I would also like to know how to do this.
anyone?

**Comment by JFossey** [comment]
I agree, it would should be configurable but I wounder how it would affect asset management.

**Comment by taiyeoguns** [comment]
+1

**Comment by gabyrusu** [comment]
you can move them changind the theme/config.php like so:
'themeDir' => '../resources/themes',
but this will affect asset manager because it will try to fetch the asset from: "/../resources/themes/default/assets/".

anyone with a solution?

**Comment by kurorido** [comment]
Hi, I create a pull request for this #122

**Comment by ttimot24** [comment]
Hi!

Is it possible to render the themes from /themes dir, a level up from public (aka laravel's root dir)?
If a change the themeDir I get a file not found exception. (config.php)
